spec/integration/transpiler_spec.rb

```diff
@@ -47,12 +47,18 @@
   }
   PLAN
 
-  it 'transpiles a yaml plan' do
+  it 'transpiles a YAML plan from a path' do
```

Comment:
Can this also include a quick test for transpiling by name?

Reply:
I added a test below this one that converts by name.

Reply:
https://github.com/puppetlabs/bolt/blob/89278e92f48ae0b7ca92ee9025fcc9a9bd198871/spec/integration/transpiler_spec.rb#L56